Dockerfile

```diff
@@ -1,6 +1,9 @@
 FROM ubuntu:20.04
 RUN apt-get update && apt-get upgrade --yes && apt-get install git python3.8 python3-pip --yes
 COPY . .
+# need to use typing-extensions<4 to deal with issue 387 bug: Docker will not build
+# todo: remove this line when 'trie' supports typing-extensions>4
```

Comment:
This is a little misleading: https://github.com/ethereum/py-trie/blob/master/setup.py#L54
I am not exactly sure what the whole deal is though..

Reply:
2.0.0 isn't released yet

https://pypi.org/project/py-trie/

Reply:
v2.0.0 is released... weird
https://pypi.org/project/trie/

Reply:
Maybe just say `TODO: Figure out a better solution or wait for it to resolve itself`.
I think this is a good patch for an immediate fix but we still ned to figure out what exactly is going on.

Reply:
@fubuloubu it could be that the dependency that we are using which uses `trie` as a dependency does not have their pin updated?

Reply:
That's definitely the case!

This is the dependency that uses it, however they updated it for v0.5.0-alpha.2:
https://github.com/ethereum/py-evm/commit/c8e5358cc33602f777037d64f94ec7081c98123c

`eth-tester` is not up to date yet tho:
https://github.com/ethereum/eth-tester/blob/master/setup.py#L26

Reply:
ok so I'll make an issue that has as a blocker that eth-tester updates

Reply:
ok so I'll make an issue that has as a blocker that eth-tester updates

Reply:
Created this to track: https://github.com/ethereum/eth-tester/pull/229

Reply:
@fubuloubu this one is also tracking in our repo so I updated it https://github.com/ApeWorX/ape/issues/316